README.md

```diff
@@ -35,84 +35,56 @@ import "github.com/swaggo/files" // swagger embed files
 ```
 
 ### Canonical example:
-
+Now assume you have implemented a simple api as following:
 ```go
-package main
-
-import (
-	"github.com/gin-gonic/gin"
-	swaggerFiles "github.com/swaggo/files"
-	ginSwagger "github.com/swaggo/gin-swagger"
-
```

Comment:
please remove the import alias.

Reply:
It seems unavoid using those import alias. Both `gin-swagger` and `files` cannot used directly.